### DISCUSSION TITLE
제페토 유니티 실행 안됨ㅠㅠ

### DISCUSSION THREAD

**Discussion by sabe31** [naverz/zepeto-studio-kor#824] [5. Q&A]
<img width="1244" alt="제페토 유니티 안 들어가짐" src="https://user-images.githubusercontent.com/106575405/171098853-fa99fc12-ab55-4205-a2b9-ec9a3e1d7848.png">
안녕하세요. 저 월드 개발을 한번 도전해 보고 싶은데 제페토 템플릿을 등록하는 것 까지는 했는데 다시 유니티에 들어갔는데 계속 화면이 이렇게 뜨는데 혹시 제가 무슨 실수를 했는지 알려주실 수 있나요ㅠㅠ

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요, Directory Not Found 에러이며 한글 폴더 이름이 깨지는 것으로 보아
폴더 이름을 영어로 하신 후에 다시 생성해 보시겠어요?

**Reply by sabe31** [reply]
감사합니다.ㅠㅠ
